bin/pt-show-grants

```diff
@@ -2433,6 +2467,16 @@ type: array
 
 Only show grants for this comma-separated list of users.
 
+=item --convert-MariaDB
+
+When set it convert some of the proprietary MariaDB syntax into valid MySQL form
+
+=item --print_identified_with_as_hex
```

Comment:
According to the user manual at https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_print_identified_with_as_hex:

> Hash values that do not contain unprintable characters still display as regular string literals, even with this variable enabled. 

Do we really need this option or checking for correct MySQL version is enough?